app-modules/service-management/src/Filament/Resources/ServiceRequestTypeResource/Pages/EditServiceRequestTypeNotifications.php

```diff
@@ -62,128 +61,72 @@ public function form(Form $form): Form
-                            ->label('Email'),
-                        Toggle::make('is_auditors_service_request_resolved_notification_enabled')
-                            ->label('Notification'),
-                        Heading::make()
-                            ->two()
-                            ->content('Customers'),
-                        Heading::make()
-                            ->three()
-                            ->content('Service Request Created'),
-                        Toggle::make('is_customers_service_request_created_email_enabled')
-                            ->label('Email'),
-                        Toggle::make('is_customers_service_request_created_notification_enabled')
-                            ->label('Notification'),
-                        Heading::make()
-                            ->three()
-                            ->content('Service Request Assigned'),
-                        Toggle::make('is_customers_service_request_assigned_email_enabled')
-                            ->label('Email'),
-                        Toggle::make('is_customers_service_request_assigned_notification_enabled')
-                            ->label('Notification'),
-                        Heading::make()
-                            ->three()
-                            ->content('Service Request Update'),
-                        Toggle::make('is_customers_service_request_update_email_enabled')
-                            ->label('Email'),
-                        Toggle::make('is_customers_service_request_update_notification_enabled')
-                            ->label('Notification'),
-                        Heading::make()
-                            ->three()
-                            ->content('Service Request Status Change (Non-Closed)'),
-                        Toggle::make('is_customers_service_request_status_change_email_enabled')
-                            ->label('Email'),
-                        Toggle::make('is_customers_service_request_status_change_notification_enabled')
-                            ->label('Notification'),
-                        Heading::make()
-                            ->three()
-                            ->content('Service Request Closed'),
-                        Toggle::make('is_customers_service_request_closed_email_enabled')
-                            ->label('Email'),
-                        Toggle::make('is_customers_service_request_closed_notification_enabled')
-                            ->label('Notification'),
-                    ]),
+                        ViewField::make('settings')
+                            ->rules(['array'])
+                            ->view('service-management::filament.resources.service-request-type-resource.pages.edit-service-request-type-notifications.matrix'),
+                    ])
+                    ->visible(ServiceRequestClosedNotificationColumns::active())
```

Comment:
I think this is fine since no one is currently actively using this. But in the future we would have to make sure this was more backwards compatible.